src/service/packet_router.proto

```diff
@@ -27,7 +27,7 @@ message packet_router_packet_report_v1 {
   uint32 payload_size = 11;
   bool free = 12;
   packet_type type = 13;
-  uint64 hpr_timestamp_ms = 14;
+  uint64 received_timestamp_ms = 14;
```

Comment:
`received_timestamp` and add a comment above it indicating the type like mentioned in the suggestions